tools/pctools/venv_inst.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# COPY and RUN this script in a container image, to:
+# create a python virtualenv named as $1,
+# pip install the modules provided as parameters from $2 on,
+# collect the resulting dependencies in a resource_$1.txt file
+# copy the txt in a common dir.
+# --- 
+# $1 CLI Cloud type: aws, azure, ec2, openstack
+# $2 resources output file
+# $3,... : install modules
+# ---
+
+echo $0 $*
+
+# parameters check
+[ $# -lt 3 ] && exit 1
+
+# create venv
+CLOUD=$1
+FILE=$2
```

Comment:
Perhaps giving the resources output file a more descriptive variable name might increase the readability?

```suggestion
OUTPUT=$2
```